src/modules/StorageProposal.rb

```diff
@@ -4475,6 +4478,9 @@ def modify(partitions)
               size_limit_k = 1024 * opts["root_base"]
               if volume["size_k"] >= size_limit_k
                 volume["userdata"] = { "/" => "snapshots" }
+                Builtins.y2milestone( "Adding userdata '/' => 'snapshots'" )
```

Comment:
Same that above. In addition, spaces after the "(" are against the Ruby coding standard.